src/org/parosproxy/paros/security/SslCertificateService.java

```diff
@@ -63,7 +64,32 @@
 	 * @throws UnrecoverableKeyException
 	 * @throws MissingRootCertificateException when it wasn't initialized.
 	 */
-	KeyStore createCertForHost(String hostname)
+	default KeyStore createCertForHost(String hostname)

```

Comment:
This should be the other way around, the default should be the new method which would call the old method (older implementations wouldn't have the new method).